cilium-cli/cli/cmd.go

```diff
@@ -39,7 +39,7 @@ func NewCiliumCommand(hooks api.Hooks) *cobra.Command {
 				return nil
 			}
 			switch cmd.Name() {
-			case "completion", "help":
+			case "completion", "help", "summary":
```

Comment:
nit: is this really necessary? Isn't the new sub-command `cilium features summary`?

Reply:
Yes, it's odd, I tried it out but it doesn't work for soem reason :confused: